mtk-automation/src/contract_actions/reward.rs

```diff
@@ -1,18 +1,17 @@
 //! Reward Module
 //!
 //! Module for reward control in the contract
-use soroban_sdk::{Env, Symbol, Map, AccountId};
-use soroban_auth::Signature;
-use crate::contract_actions::member;
 use crate::contract_actions::datakey::DataKey;
-use crate::contract_actions::identifier_wrapper;
+use crate::contract_actions::member;
 use crate::contract_actions::token_operation;
+use soroban_sdk::Address;
+use soroban_sdk::{Env, Map, Symbol};
```

Comment:
The `Address` struct can be imported on the next line
```suggestion
use soroban_sdk::{Env, Map, Symbol, Address};
```